language/api/analyze.py

```diff
@@ -25,13 +25,20 @@
 from oauth2client.client import GoogleCredentials
 
 
+# TODO REMOVE - when discovery is public
+DISCOVERY_URL = ('https://language.googleapis.com/$discovery/rest?'
+                 'version=v1&labels=GOOGLE_INTERNAL')
+
+
 def get_service():
     credentials = GoogleCredentials.get_application_default()
     scoped_credentials = credentials.create_scoped(
         ['https://www.googleapis.com/auth/cloud-platform'])
     http = httplib2.Http()
     scoped_credentials.authorize(http)
-    return discovery.build('language', 'v1beta1', http=http)
+    return discovery.build('language', 'v1',
+                           http=http,
```

Comment:
Have you tried recently? I think there was a change to the api client recently (possibly made by Jon) that retried getting the discovery doc if it gets a permission-denied error.